client/my-sites/themes/controller/index.node.js

```diff
@@ -69,9 +69,8 @@ export function details( context, next ) {
 	const { slug } = context.params;
 	const user = getCurrentUser( context.store.getState() );
 	const themeName = ( getThemeDetails( context.store.getState(), slug ) || false ).name;
-	const title = i18n.translate( '%(theme)s Theme', {
-		args: { theme: themeName },
-		textOnly: true
+	const title = i18n.translate( '%(themeName)s Theme', {
+		args: { themeName: themeName }
```

Comment:
You could shorten that to `args: { themeName }`, see https://github.com/lukehoban/es6features#enhanced-object-literals.